src/altair/epoch_processing.rs

```diff
@@ -102,8 +182,15 @@ pub fn process_participation_flag_updates<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
 ) -> Result<()> {
+    let current_participation = mem::take(&mut state.current_epoch_participation);
+    state.previous_epoch_participation = current_participation;
+    // @dev intention is to set `rotate_participation` to a `List` of u8 values of length state's validators
+    // But, if `current_epoch_participation` is of length `VALIDATOR_REGISTRY_LIMIT` -- does this pose an issue?
+    let rotate_participation = vec![ParticipationFlags::default(); state.validators.len()];
+    state.current_epoch_participation = rotate_participation
+        .try_into()
+        .expect("should convert from Vec to List");
```

Comment:
can't we just do this ?
```suggestion
    state.current_epoch_participation = Default::default();
```